tools/test-backend

```diff
@@ -69,20 +70,9 @@ not_yet_fully_covered = [
     "analytics/views/stats.py",
     "analytics/views/support.py",
     # Major lib files should have 100% coverage
-    "zerver/lib/addressee.py",
     "zerver/lib/markdown/__init__.py",
-    "zerver/lib/cache.py",
-    "zerver/lib/cache_helpers.py",
-    "zerver/lib/i18n.py",
-    "zerver/lib/email_notifications.py",
```

Comment:
moved to [PR 22060](https://github.com/zulip/zulip/pull/22060) incl replacing a bunch of @zulip.com's